src/main/resources/assets/templates/letters/chips/CSIDVDEFLET_content_v1.html

```diff
@@ -0,0 +1,109 @@
+<p class="subject-line-spacer"></p>
+
+<table>
+    <tr>
+        <td class="verification-requirements">
+            <p class="verification-requirements-heading">1. Verify your identity</p>
+            <p class="verification-requirements-text">For information on how to do this, search
+                online for ‘Verifying your identity for
+                Companies House’, or visit:
+                <span class="emphasis">gov.uk/guidance/verifying-your-identity-for-companies-house</span>
+            </p>
+            <p class="verification-requirements-text">After you’ve verified your identity, you’ll get a
+                unique identifier, called a Companies House
+                personal code.</p>
+            <p class="verification-requirements-text">If you already have a personal code, you do not
+                need to verify your identity again.</p>
+            <br/>
+            <p class="emphasis">All registered company directors will need
+                a personal code</p>
+        </td>
+        <td class="verification-requirements-spacer"></td>
+        <td class="verification-requirements">
+            <p class="verification-requirements-heading">2. Provide your Companies
+                House personal code</p>
+            <p class="verification-requirements-text">This is so we can connect your verified identity
+                to the director name on our records.</p>
+            <p class="verification-requirements-text">Search online for the ‘Provide identity
+                verification details for a person with significant control’ service, or visit:<br/>
+                <span class="emphasis">[insert link]</span>
+            </p>
+            <p class="verification-requirements-text">You’ll also be asked to tick a statement
+                confirming that you’ve verified your identity
+                when filing the companies confirmation
+                statement.</p>
+            <p class="verification-requirements-text">You need to do this even if you’ve
+                already done it for other roles you have at this
+                company, or at other companies.</p>
+        </td>
+    </tr>
+</table>
+
+<p class="small-heading">Exceptional circumstances</p>
+<p>If you are unable to comply due to exceptional circumstances, you may submit written representations by
+    [insert contact method] within <span class="emphasis">28 days</span> of the date of this letter.</p>
+
+<p>If the exceptional circumstances are accepted, enforcement action will be paused for a limited period to
+    allow time for you to submit the required documents. If enforcement action is paused, the company and its
+    directors remain in default until the required documents are received. If the documents are not submitted
+    by the deadline we provide, enforcement action will resume without further notice.</p>
```

Comment:
Very minor but do we need the line wrap? Wouldn't maintenance be easier if we didn't wrap it?